### ISSUE TITLE
CLOSED

### ISSUE THREAD

**Issue by doubitcoin** [sammy007/open-ethereum-pool#371]
hi how to run this config json?
so i not see folder with "./build/bin/open-ethereum-pool"
how to i can start this config?
thanks

**Comment by mikeyb** [comment]
[READ](https://github.com/sammy007/open-ethereum-pool/blob/master/README.md)

**Comment by doubitcoin** [comment]
yes I have done everything, but did not find the .build / open-ethereum-pool/bin/config.json folder, and the config.json file is in the open-ethereum-pool / config.json folder

**Comment by mikeyb** [comment]
Read it a few more times until you get a build directory, its literally one of the first things you should have done

**Comment by mikeyb** [comment]
And you wont get further help til you submit a proper issue.  Remember that big chunk of text you deleted when you opened this issue?  WELL DONT DELETE IT

#### System information

Geth version: `geth version` or commit hash if `develop`

Redis version: `redis-server --version`

OS & Version: Linux/OSX

Commit branch and hash: (`master`, `ba60e1c`)

#### Expected behaviour


#### Actual behaviour


#### Steps to reproduce the behaviour


#### Backtrace

````
[backtrace]
````

**Comment by mikeyb** [comment]
close issue
